# Goal
string robot_name
geometry_msgs/PoseStamped pose
bool pick
geometry_msgs/Transform offset
geometry_msgs/Transform approach_offset
geometry_msgs/Transform departure_offset
float32 speed_fast
float32 speed_slow
---
# Result
int32 SUCCESS=0
int32 MOVE_FAILURE=1
int32 APPROACH_FAILURE=2
int32 GRASP_FAILURE=3
int32 DEPARTURE_FAILURE=4
int32 PREEMPTED=10

int32 result
---
# Feedback
int32 IDLING=0
int32 MOVING=1
int32 APPROACHING=2
int32 GRASPING_OR_RELEASING=3
int32 DEPARTING=4

int32 stage
